publiccode.yml

```diff
@@ -8,11 +8,17 @@ categories:
 description:
   en:
     features:
-      - Python Template
-    genericName: python_project
+      - AI
+      - Validation
+      - Transparency
+      - Algorithm
+    genericName: tad
     longDescription: |
-      This a an example python template project.
-    shortDescription: Python Template project
+      TAD is a modern tool to apply technical and non-technical tests for an AI model.
```

Comment:
Maybe  refer here to the index.md of TAD in ai-validation?

Reply:
This is a stand alone file ready by scrappers. I do not think it has logic to scrape the README.md. Do you have any examples of that so i can implement it?

Reply:
I don't have any example unfortunately, just wanted to pick your brain on this point as I wanted to minimize the amount of duplicate stuff we have. But in handsight I think it would be a little bit overengineering to scrape the README.md, what about we put the link to the github? https://minbzk.github.io/ai-validation/Projects/TAD/

Reply:
Solved